terraform/etl/49-lambda-streetscene-ingestion.tf

```diff
@@ -126,6 +126,29 @@ resource "aws_iam_role_policy_attachment" "streetscene_street_systems_lambda_sec
   policy_arn = aws_iam_policy.streetscene_street_systems_lambda_secret_access[0].arn
 }
 
+# Define a IAM Policy Document for Glue StartCrawler Permissions:
+data "aws_iam_policy_document" "streetscene_street_systems_glue_crawler" {
+  statement {
+    actions   = ["glue:StartCrawler"]
+    effect    = "Allow"
+    resources = ["*"]
+  }
+}
+
+# create a New IAM Policy Resource:
+resource "aws_iam_policy" "streetscene_street_systems_glue_crawler" {
+  count  = local.create_street_systems_resource_count
+  name   = "streetscene_street_systems_glue_crawler_access"
+  policy = data.aws_iam_policy_document.streetscene_street_systems_glue_crawler.json
+}
+
+# attach the gov_notify_glue_crawler to the housing_gov_notify_ingestion_lambda_role by creating a new aws_iam_role_policy_attachment resource.
```

Comment:
Your lambda role called `streetscene_street_systems_lambda_assume_role`, you can remove "housing_gov_notify_ingestion_" this part of the comments, generally call it `lambda_role`.

Reply:
I've updated the comments